docs/v3/core/index.rst

```diff
@@ -780,29 +782,48 @@ above, but using a (currently made up) extension data type::
 Group metadata
 --------------
 
+Mandatory
+^^^^^^^^^
+
 A Zarr group metadata object must contain the following mandatory key:
 
 ``zarr_format``
-^^^^^^^^^^^^^^^
+"""""""""""""""
 
     An integer defining the version of the storage specification to which the
     array store adheres, must be ``3`` here.
 
 ``node_type``
-^^^^^^^^^^^^^^^
+"""""""""""""""
 
     A string defining the type of hierarchy node element, must be ``group``
```

Comment:
```suggestion
    A string denoting the type of hierarchy node element. It must be the string ``"group"``
```

Reply:
I'm skipping comments which are not extension related and are outside of the changes proposed here. (Marked with 👀 to note that I've reviewed them)